libsolidity/analysis/ControlFlowBuilder.cpp

```diff
@@ -287,6 +298,21 @@ bool ControlFlowBuilder::visit(FunctionCall const& _functionCall)
 				m_currentNode = nextNode;
 				return false;
 			}
+			case FunctionType::Kind::Internal:
+			{
+				m_currentNode->functionCalls.emplace_back(&_functionCall);
+				solAssert(m_revertNode, "");
+				visitNode(_functionCall);
+				_functionCall.expression().accept(*this);
+				ASTNode::listAccept(_functionCall.arguments(), *this);
+
+				auto nextNode = newLabel();
+				connect(m_currentNode, nextNode);
+				m_currentNode = nextNode;
+
+				return false;
```

Comment:
```suggestion
```

Reply:
Or actually is it important that we split the node here?

Reply:
It at least used to be. It seems with the current updates in the PR it no longer is needed.